src/Copy/Text.elm

```diff
@@ -183,7 +183,7 @@ t key =
             "Tomorrow"
 
         EventsFilterLabelAll ->
-            "All Events"
+            "All Future Events"
```

Comment:
```suggestion
            "All future events"
```

This fits the grammar of the rest of the site better from my cursory look over. Please ignore if that's not correct.